### PR TITLE
Use the right setuptools_scm version

### PR DESCRIPTION
**Pull request by pkoch**
Unsure if this is the right call, but I want to see it run.

### CODE REVIEW

**Comment by pkoch** [comment]
@fbidu can you take this over from me? 😭 You seem to have some millage with this, and I feel totally in the dark.

**Comment by fbidu** [comment]
> can you take this over from me? sob You seem to have some millage with this, and I feel totally in the dark.

@pkoch sure!

**Comment by fbidu** [comment]
@pkoch can you make me admin or something of this project so I can push things to your branch?

**Comment by pkoch** [comment]
Gave @Doist/backend maintainer status. lmk if that's not enough.

**Comment by fbidu** [comment]
@pkoch it was! Solving this took a bit more elbow grease than what I expected due to mypy behaving funny, but it worked. Thanks! I'll merge this now

File: tox.ini [hunk 1]
```diff
@@ -19,4 +19,5 @@ commands =
 deps =
     mypy
 commands =
+    python3 -m pip install types-pytz
```
Comment (Schnouki) [inline tox.ini:22]: Hmm, can't this be moved in `deps`, 2 lines above?

**Comment by pkoch** [comment]
Thank you so much for untangling this mess, @fbidu! <3 One less broken window! 💪

File: tox.ini (hunk 1 above)
Comment (pkoch) [reply tox.ini:22]: Addressing that here: https://github.com/Doist/python-timezones/pull/18

Comment (pkoch) [reply tox.ini:22]: Now on master. Thanks for helping us picking up the litter! :)

**Pull request merged**